.github/workflows/src/arm-incremental-typespec.js

```diff
@@ -30,7 +31,7 @@ export default async function incrementalTypeSpec({ core }) {
   const changedFiles = await getChangedFiles(options);
 
   // Includes swaggers, readmes, and examples
-  const changedRmFiles = changedFiles.filter(resourceManager);
+  const changedRmFiles = changedFiles.filter((f) => specification(f) && resourceManager(f));
```

Comment:
`specification()` and `resourceManager()` are decoupled, so callers must include both if they want both